### PR TITLE
WINC-749: Document dual NIC limitation

### PR DESCRIPTION
**Pull request by jrvaldes**
This PR documents the dual NIC limitation. In addition, fixes incorrectly formatted tables.

### CODE REVIEW

**Comment by selansen** [comment]
Can you update the title and description with "multiple NIC support"? 
This will be easy to search for PR in the future.  Basically, this is about multiple NIC support.

File: docs/wmco-prerequisites.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ these errors, only use the appropriate version according to the cloud provider i
 The following tables outline the type of networking configuration and Windows Server versions to be used based on your 
 cloud provider. The network configuration must be completed during the installation of the cluster.
   
-Note: OpenShiftSDN networking is the default network for OpenShift clusters. It is NOT supported by WMCO.
+Note: 
+* OpenShiftSDN networking is the default network for OpenShift clusters. It is NOT supported by WMCO.
+* Multiple networks (dual NICs) is NOT supported by WMCO.
```
Comment (selansen) [inline docs/wmco-prerequisites.md:34]: This is not related to multiple networks. This is about dual NIC. There are ways to connect same NIC with two different subnets ( IPs).  Let's just make a note saying 
Dual NIC is not supported by WMCO.

**Review by selansen — CHANGES_REQUESTED** [review]

File: docs/wmco-prerequisites.md (hunk 1 above)
Comment (mburke5678) [reply docs/wmco-prerequisites.md:34]: @jrvaldes @selansen Do we know if dual NICs are supported by OpenShift? I don't see that phrase in our docs. If we go with _Dual NIC is not supported by WMCO._, folks could assume that OCP does support dual NIC.

Comment (jrvaldes) [reply docs/wmco-prerequisites.md:34]: @mburke5678   I tend to agree with you. That's why I went with `Multiple networks` instead of `Dual NIC`.

Comment (jrvaldes) [reply docs/wmco-prerequisites.md]: Updated to `Dual NIC is not supported by WMCO` as per @selansen  request.

Comment (selansen) [reply docs/wmco-prerequisites.md]: @mburke5678 I read in OpenShift docs and  learnt they do support dual (Multiple ) NIC on the Linux side.
Pls check this one.
https://www.redhat.com/en/blog/multus-takes-leading-role-container-networking

**Review by selansen — APPROVED** [review]
/lgtm

**Comment by jrvaldes** [comment]
> @jrvaldes: #980 failed to apply on top of branch "community-4.9":
> 
> ```
> Applying: Fix incorrectly formatted tables
> Using index info to reconstruct a base tree...
> M	docs/wmco-prerequisites.md
> Falling back to patching base and 3-way merge...
> Auto-merging docs/wmco-prerequisites.md
> CONFLICT (content): Merge conflict in docs/wmco-prerequisites.md
> error: Failed to merge in the changes.
> hint: Use 'git am --show-current-patch=diff' to see the failed patch
> Patch failed at 0001 Fix incorrectly formatted tables
> When you have resolved this problem, run "git am --continue".
> If you prefer to skip this patch, run "git am --skip" instead.
> To restore the original branch and stop patching, run "git am --abort".
> ```

Depends on https://github.com/openshift/windows-machine-config-operator/pull/999

**Comment by jrvaldes** [comment]
/cherry-pick community-4.9

https://github.com/openshift/windows-machine-config-operator/pull/999 has merge
